### DISCUSSION TITLE
Interactive map does not show up

### DISCUSSION THREAD

**Discussion by ElKa01** [opengeos/leafmap#49] [Q&A] [answered]
Hello. I followed the "Get started" instructions step-by-step and could plot an interactive map. Then I clicked through the notebook-series and tried to plot a split map following the instructions. However, this message showed up in Jupyter notebook: 
![image](https://user-images.githubusercontent.com/86201230/122682651-14bf2600-d1fb-11eb-93c1-b40025c564d0.png)

So I tried to replot the beginning tutorial, but now it did not work anymore: 
![image](https://user-images.githubusercontent.com/86201230/122682673-37e9d580-d1fb-11eb-946b-a2043830d4fc.png)

In the FAQ I looked at the "Why the interactive map does not show up" and installed the two commands within the leafmap conda environment. 

Then I created a new envirnoment and searched in different forums about issues when trying to plot ipyleaflet. Finally I restarted my computer, but without success so far.

Thanks for your help in advance.

**Comment by giswqs** [comment, marked as answer]
Check out this video tutorial for installing leafmap: https://youtu.be/-UPt7x3Gn60

```
    conda create -n geo python=3.8
    conda activate geo
    conda install geopandas
    conda install mamba -c conda-forge
    mamba install leafmap xarray_leaflet -c conda-forge
```

**Reply by ElKa01** [reply]
Thank you for the fast reply. Unfortunately it did not work. I created a new environment, exactly the way it shows in the tutorial, but the map is still not shown. 
However, the solution shown in this page worked for me: Manually enable the ipywidgets notebook extension with
_jupyter nbextension enable --py widgetsnbextension_
https://ipywidgets.readthedocs.io/en/stable/user_install.html
